frontend/src/Components/CustomDivider/CustomDivider.scss

```diff
@@ -1,21 +1,28 @@
+@import "Styles/Colours.scss";
+@import "Styles/Variables.scss";
+@import "Styles/MediaQueries.scss";
+
 .divider {
   background-image: url("../../Assets/images/divider.png");
   background-position: center;
   background-repeat: no-repeat;
   background-size: cover;
+
   width: 100%;
   height: 90px !important;
 }
 
 .filter-gradient {
-  background-image: linear-gradient(to top, transparent 0%, black 90%);
-  height: 100% !important;
-  width: 100% !important;
-}
+  .divider-text {
+    font-family: "Roboto condensed", "Helvetica", "Arial", "sans-serif";
```

Comment:
We should maybe set the fonts globally

Reply:
You right, used `$font-family-condensed`